flutter/lib/src/integrations/native_app_start_integration.dart

```diff
@@ -101,12 +106,37 @@ class NativeAppStartIntegration extends Integration<SentryFlutterOptions> {
           return;
         }
 
+        List<TimeSpan> nativeSpanTimes = [];
+        for (final entry in nativeAppStart.nativeSpanTimes.entries) {
+          try {
+            final startTimestampMs =
+                entry.value['startTimestampMsSinceEpoch'] as int;
+            final endTimestampMs =
+                entry.value['stopTimestampMsSinceEpoch'] as int;
+            nativeSpanTimes.add(TimeSpan(
+              start: DateTime.fromMillisecondsSinceEpoch(startTimestampMs),
+              end: DateTime.fromMillisecondsSinceEpoch(endTimestampMs),
+              description: entry.key as String,
+            ));
+          } catch (e) {
+            // ignore: invalid_use_of_internal_member
+            _hub.options.logger(
+                SentryLevel.warning, 'Failed to parse native span times: $e');
+            continue;
+          }
+        }
+
+        // We want to sort because the native spans are not guaranteed to be in order.
+        // Performance wise this won't affect us since the native span amount is very low.
+        nativeSpanTimes.sort((a, b) => a.start.compareTo(b.start));
```

Comment:
also happy to hear other recommendations to solve this if this is not sufficient enough

Reply:
guess it should be enough